src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,232 @@
-              <IconButton
-                aria-label="Home"
-                className={classes.roundButton}
-                color={key === '' ? 'primary' : 'secondary'}
-                data-test="home-button"
-                size="large"
-              >
-                <Home />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/people`} passHref>
-              <IconButton
-                aria-label="People"
-                className={classes.roundButton}
-                color={key.startsWith('/people') ? 'primary' : 'secondary'}
-                data-test="people-button"
-                size="large"
-              >
-                <People />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/journeys`} passHref>
-              <IconButton
-                aria-label="Journeys"
-                className={classes.roundButton}
-                color={key.startsWith('/journeys') ? 'primary' : 'secondary'}
-                data-test="people-button"
-                size="large"
-              >
-                <Explore />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/areas`} passHref>
-              <IconButton
-                aria-label="Areas"
-                className={classes.roundButton}
-                color={key.startsWith('/areas') ? 'primary' : 'secondary'}
-                data-test="area-button"
-                size="large"
-              >
-                <Map />
-              </IconButton>
+              <Avatar
+                alt="icon"
+                src={`/api/orgs/${orgId}/avatar`}
+                sx={{ alignSelf: 'center', marginLeft: 0.5 }}
+              />
             </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/projects`} passHref>
-              <IconButton
-                aria-label="Projects"
-                className={classes.roundButton}
-                color={key.startsWith('/projects') ? 'primary' : 'secondary'}
-                data-test="calendar-button"
-                size="large"
-              >
-                <Event />
-              </IconButton>
-            </NextLink>
-          </ListItem>
+          )}
+          {open && (
+            <ZUIFuture future={model.getOrganization(orgId)}>
+              {(data) => {
+                return (
+                  <Box alignSelf="center">
+                    <Box display="flex" justifyContent="start">
+                      {hover ? (
+                        <IconButton onClick={toggleDrawer}>
+                          <ChevronLeftIcon
+                            onClick={handleDrawerToggle}
```

Comment:
Here you have double mouse event handlers again. I'm sorry I didn't catch this in the first review. Removing the inner one (and deleting the unused function `handleDrawerToggle`) seems to work fine.